lib/my_zipcode_gem.rb

```diff
@@ -1,3 +1,4 @@
+require 'rails/generators/actions'
```

Comment:
Interesting. I actually prefer single quotes unless you need to avoid escaping a single quote or you are interpolating.